lib/transaction.py

```diff
@@ -166,47 +166,6 @@ def short_hex(bytes):
     return t[0:4]+"..."+t[-4:]
 
 
-def script_GetOp(_bytes):
-    i = 0
-    blen = len(_bytes)
-    while i < blen:
-        vch = None
-        opcode = _bytes[i]
-        i += 1
-
-        if opcode <= opcodes.OP_PUSHDATA4:
-            nSize = opcode
-            if opcode == opcodes.OP_PUSHDATA1:
-                nSize = _bytes[i] if i < blen else 0
-                i += 1
-            elif opcode == opcodes.OP_PUSHDATA2:
-                (nSize,) = struct.unpack_from('<H', _bytes, i) if i+2 <= blen else (0,) # tolerate truncated script
-                i += 2
-            elif opcode == opcodes.OP_PUSHDATA4:
-                (nSize,) = struct.unpack_from('<I', _bytes, i) if i+4 <= blen else (0,)
-                i += 4
-            vch = _bytes[i:i + nSize] # array slicing here never throws exception even if truncated script
-            i += nSize
-
-        yield opcode, vch, i
```

Comment:
interestingly, the third part of this tuple never got used anywhere (and indeed it's not really good for anything).

Reply:
:)